server/index.js

```diff
@@ -8,7 +8,7 @@ import schema from './schema';
 
 dotenv.config();
 
-const port = process.env.PORT || process.env.port; 
```

Comment:
I did suggest defaulting to a value and also reading from an environment variable if available.

const port = process.env.PORT || 8080;

Reply:
Oh now I get, so reading the port number as default value than to read it from env variable.